src/perf/lib/PerfClient.cpp

```diff
@@ -144,6 +144,9 @@ PerfClient::Init(
     // General configuration options
     //
 
+    #ifndef _KERNEL_MODE
+    TryGetValue(argc, argv, "qtip", &UseQtip);
+    #endif // _KERNEL_MODE
```

Comment:
Please update help text and docs for secnetperf for this change. Let's also put this after `tcp` and remove the ifdefs here (keep them in the help text).

Reply:
Updated helptext to display the correct/valid io modes.